app/Commands/Launch.php

```diff
@@ -36,14 +36,37 @@ public function handle()
         $appName = $this->ask("Choose an app name (leave blank to generate one)");
         if (!$appName) $appName = "--generate-name"; //not putting this as the default answer in $this->ask so '--generate-name' is not displayed in the prompt
 
-        $result = Process::run("flyctl apps create -o personal --machines $appName");
+//        $result = Process::run("flyctl apps create -o personal --machines $appName");
```

Comment:
@Johannes-Werbrouck the creation logic is commented out, this is fine to merge right?

Reply:
I commented it out so I wouldn't have to delete heaps of apps manually. I shouldn't have left it in but it might be useful to you :)

Reply:
yup useful! merging this now